README.md

```diff
@@ -0,0 +1,22 @@
+# Additional Fruits
+This module adds various different fruit blocks and includes 3 different stages for each plant.
+
+**Types of Fruits:**
+* Blueberry
+* Bushes
+  - Cranberry
+  - Gojiberry
+  - Guava
+  - Peach
+* Gooseberry
+* Raspberry
+* Strawberry
+
+
+How to Contribute
+ ============
+ 1. Fork and clone this repository locally
+ 2. To add another **fruit block** or **fruit bush**, create a new prefab
```

Comment:
This could use a bit more detail since each fruit at present involves multiple blocks, block tiles, and prefabs. What goes into each? Chances are the SimpleFarming system describes that already so it may be simpler to link there for further instructions.